knexfile.js

```diff
@@ -0,0 +1,35 @@
+require('dotenv').config({
+  path: './.env'
+});
+
+module.exports = {
+  development: {
+    client: 'pg',
+    connection: {
+      database: process.env.POSTGRES_DB,
```

Comment:
To be consistent with you rnaming convention for the test DB, you could change this to POSTGRES_DEV_DB

Reply:
Done!